api_docs/swagger/paths/partner_orders/pre_activation.yaml

```diff
@@ -0,0 +1,22 @@
+post:
+  tags:
+    - partner_orders
+  parameters:
+    - $ref: '/swagger/swagger.yaml#/components/parameters/Date'
+    - $ref: '/swagger/swagger.yaml#/components/parameters/ContentMd5'
+    - $ref: '/swagger/swagger.yaml#/components/parameters/Authorization'
+  summary: Place a pre-activated OEM partner order
```

Comment:
Do we explain somewhere the difference between a "pre-activated" partner order and "regular" (?) partner order?